cmd/fleet/main.go

```diff
@@ -215,13 +215,13 @@ func newFleetService(auth mainflux.AuthServiceClient, db *sqlx.DB, logger *zap.L
 			Subsystem: "api",
 			Name:      "request_count",
 			Help:      "Number of requests received.",
-		}, []string{"method"}),
+		}, []string{"method", "owner_id", "service", "id"}),
```

Comment:
So I think best practice is to not put "service" here since it will never change (that can be added by the scraper). Also, I believe we should use "thing_id" instead of "id" here. This way we can clearly specify during queries which ID we mean, and it results in lower cardinality for the label since other services also have "id"